### PR TITLE
New license request: ASWF Digital Assets License 1.0

### PR DESCRIPTION
**Pull request by BassCoder2808**
- Solves #1550 
- Added the relevant xml and txt file for the ASWF Digital Assets License 1.0

### CODE REVIEW

**Comment by BassCoder2808** [comment]
Hi @jlovejoy, I have created the PR, can you please let me know if anything else needs to be changed from my end. Also a job was failing for this, but I was not able to figure out, exactly whats the error, can you also please help me in that.

**Comment by jlovejoy** [comment]
A couple things I see wrong (which may or may not be causing the tests to fail!):

- missing License List version tag
- no need to repeat title outside of titleText tag 
- xml uses "" in first couple lines, whereas text file and original file uses <> e.g., `License for <Asset Name> ` 

this must be the same. I just added some guidance on that last item here: https://github.com/spdx/license-list-XML/blob/main/DOCS/xml-fields.md#other-things-to-watch-out-for

**Comment by BassCoder2808** [comment]
Thanks @jlovejoy for responding back, I will make the changes which you have mentioned above

File: src/ASWF-Digital-Assets-1.0.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="ASWF-Digital-Assets-1.0"
+   name="ASWF Digital Assets License version 1.0" listVersionAdded="1.0">
```
Comment (swinslow) [inline src/ASWF-Digital-Assets-1.0.xml:4]: `listVersionAdded` is intended to be the version of the SPDX License List release in which this license is first added. The upcoming release is v3.21, so this should be "3.21".

**Review by swinslow — CHANGES_REQUESTED** [review]
@BassCoder2808 thanks for this! Just one minor change needed, see inline -- then this should be good to go, as far as I can tell from a quick look.

File: src/ASWF-Digital-Assets-1.0.xml (hunk 1 above)
Comment (BassCoder2808) [reply src/ASWF-Digital-Assets-1.0.xml:4]: Thanks @swinslow, I will make that change right away

**Comment by jlovejoy** [comment]
woo-hoo, everything is passing now!

**Comment by jlovejoy** [comment]
@swinslow @BassCoder2808 - I'm going to merge now :)

**Pull request merged**